arborist/server.go

```diff
@@ -334,20 +334,31 @@ func (server *Server) handleAuthProxy(w http.ResponseWriter, r *http.Request) {
 	authRequest.stmts = server.stmts
 	w.Header().Set("REMOTE_USER", authRequest.Username)
 
-	rv, err := authorizeUser(authRequest)
-	if err != nil {
-		msg := fmt.Sprintf("could not authorize user: %s", err.Error())
-		server.logger.Info("tried to handle auth request but input was invalid: %s", msg)
-		response := newErrorResponse(msg, 400, nil)
-		_ = response.write(w, r)
+	if (authRequest.Username == "") && (authRequest.ClientID == "") {
+		msg := "unauthorized: did not provide a username and/or client ID in request"
```

Comment:
should this be and/or, in this case it should just be and correct?

Reply:
see my other comment; providing only one of the 2 is fine, as long as at least 1 is provided